asm/sjis.s

```diff
@@ -0,0 +1,206 @@
+.include "macro.inc"
+
+# assembler directives
+.set noat      # allow manual use of $at
+.set noreorder # don't insert nops after branches
+.set gp=64     # allow use of 64-bit general purpose registers
+
+.section .text
+
+.balign 16
+
+# s32 Kanji_OffsetFromShiftJIS(s32 shiftJISCodepoint)
```

Comment:
```suggestion
# s32 Kanji_OffsetFromShiftJis(s32 shiftJisCodepoint)
```
This one is probably up for more discussion. I think that abbreviations still use the camel case for names (think Dma instead of DMA, Id instead of ID), but I'm not sure how standardized that is